provider/marathon/config.go

```diff
@@ -19,34 +19,91 @@ import (
 
 func (p *Provider) buildConfiguration() *types.Configuration {
 	var MarathonFuncMap = template.FuncMap{
-		"getBackend":                  p.getBackend,
+		"getBackend":   p.getBackend,
+		"getDomain":    getFuncStringService(label.TraefikDomain, p.Domain), // FIXME dead ?
```

Comment:
What does your comment `FIXME dead ?` mean?  Are you asking if this method is not being used?  If that's the question, it is being used, it was added for ability to expose this information to custom templates, as it is very necessary information in many cases to do so. Same goes for `getSubDomain` -
 https://github.com/containous/traefik/pull/1693